### ISSUE TITLE
Downloading only 30 seconds for each video

### ISSUE THREAD

**Issue by ivan-salamon** [22nds/treehouse_video_downloader#6]
Links and downloading is working fine but the length of every downloaded video is just 30 seconds
A bug or we got blocked by Treehouse?

Thanks

**Comment by 22nds** [comment]
Hi Ivan!
 
Check your username and password. The error indicates, they are not correct. 

Best, 
M.

**Comment by ivan-salamon** [comment]
Hi Maja,

Both are correct. If I try to download a video manually through html video source url, I get the the full length. Using a script, only 30 seconds.
Does it work only with pro account? I'm basic...

Thanks

**Comment by 22nds** [comment]
Ivan, thanks for the feedback! I have pushed the update and changed the default video format to webm. I don't have Treehouse account any more so I cannot test it, but if somebody comes across the same issue in the future, please open a new issue and we can discuss it there. 

Best,
M.

**Comment by soumikd4** [comment]
again 30 seconds video downloading

**Comment by 22nds** [comment]
> again 30 seconds video downloading

Hi @soumikd4,

Please doublecheck your username and password. 

Best,
M.

**Comment by soumikd4** [comment]
Hi M,
I have checked it.
still its not working.
30 seconds only.
regards,
Soumikd4

**Comment by soumikd4** [comment]
this is my details
First NameSoumik
Last NameDhar
Email Addresssoumikd4@gmail.com
Passwordmaximum
Profile Namesoumikdhar
Treehouse Profile URLteamtreehouse.com/soumikdhar

**Comment by soumikd4** [comment]
for ur refernce

**Comment by 22nds** [comment]
Code was updated and now works as expected. Feel free to let me know if you encounter any issue.

**Comment by soumikd4** [comment]
Sure.

**Comment by soumikd4** [comment]
One thing
I have downloaded all file before ur latest fix. Is it possible to download
subtitles separate ly?
Also one folder is empty and log says 1 library wasnt downloaded. Im
sending u links.
log file: https://teamtreehouse.com/library/javascript-and-the-dom-2
empty folder: https://teamtreehouse.com/library/what-is-html
